test/plugins/features/movie/index.test.js

```diff
@@ -20,4 +21,28 @@ describe('movies integration', () => {
 
   });
 
+  describe('list', () => {
+
+    const movie = { title: 'foo', release_year: 2016 };
+
+    beforeEach(() => {
+      return Knex('movies').insert(movie);
+    });
+
+    it('lists all movie', () => {
+      return Movies.inject({
+        url: '/movies',
+        method: 'GET',
+        payload: {}
+      })
+      .then((response) => {
+        expect(response.statusCode).to.eql(200);
+        expect(response.result.length).to.be.above(0);
+        expect(response.result[0].title).to.be.equal(movie.title);
+        expect(response.result[0].release_year).to.be.equal(movie.release_year);
```

Comment:
Since you already tested that these properties were there in your controller tests, you don't need to test them again.